src/models/topic-extraction/TopicExtractionJob.ts

```diff
@@ -0,0 +1,17 @@
+import { JobStatus } from '../JobStatus';
+
+/** Topic extraction job model */
+export interface TopicExtractionJob {
+    id: string;
+    status: JobStatus;
+    type: string;
```

Comment:
```suggestion
    type: JobType;
```
Add topic and sentiment to the job type enum